tools/astarte_export/Dockerfile

```diff
@@ -1,17 +1,27 @@
-FROM elixir:1.11.4 as builder
+FROM hexpm/elixir:1.14.5-erlang-25.3.2-debian-bullseye-20230522-slim as builder
+
+# install build dependencies
+# --allow-releaseinfo-change allows to pull from 'oldstable'
+RUN apt-get update --allow-releaseinfo-change -y && apt-get install -y build-essential git \
+    && apt-get clean && rm -f /var/lib/apt/lists/*_*
 
 WORKDIR /app
 
 # Install hex
 RUN mix local.hex --force && \
-mix local.rebar --force && \
-mix hex.info
+    mix local.rebar --force && \
+    mix hex.info
 
 # Pass --build-arg BUILD_ENV=dev to build a dev image
 ARG BUILD_ENV=prod
 
 ENV MIX_ENV=$BUILD_ENV
 
+ENV CASSANDRA_DB_HOST="127.0.0.1"
+ENV CASSANDRA_DB_PORT="9042"
+ENV REALM="test"
+ENV XML_FILE="../files/test.xml"
```

Comment:
These values should be settable by the `docker run` command; the values provided here can be sensible default but I would expect a user to be able to run the container by doing something like
```
docker run -e "CASSANDRA_DB_HOST=http://my.host.com" -e "CASSANDRA_DB_PORT=h9042" -e "REALM=myverynicerealm" -e "XML_FILE=a/b/c.xml" astarte/astarte_export
```

Reply:
I have a problem with CASSANDRA_DB_HOST and CASSANDRA_DB_PORT when providing these variables in run application. This causes a break in function for connecting to the database. As a result of not reading the environment variables.

Reply:
You can take a look at the Edgehog backend config (https://github.com/edgehog-device-manager/edgehog/blob/main/backend/config/runtime.exs) and Dockerfile (https://github.com/edgehog-device-manager/edgehog/blob/main/backend/Dockerfile) as an example on handling env vars when using Docker